src/services/localSSHService.ts

```diff
@@ -60,26 +63,30 @@ export class LocalSSHService extends Disposable implements ILocalSSHService {
     }
 
     private async initialize() {
-        let failureCode: FailedToInitializeCode | undefined;
-        const useLocalAPP = String(Configuration.getUseLocalApp());
-        try {
-            const locations = await this.copyProxyScript();
-            await this.configureSettings(locations);
-            this.isSupportLocalSSH = true;
-        } catch (e) {
-            this.logService.error(e, 'failed to initialize');
-            failureCode = 'Unknown';
-            if (e?.code) {
-                failureCode = e.code;
-            }
-            if (e?.message) {
-                e.message = `Failed to initialize: ${e.message}`;
+        const gitpodHost = this.hostService.gitpodHost;
+        const lockFile = vscode.Uri.joinPath(this.context.globalStorageUri, `${gitpodHost}`);
```

Comment:
Shouldn't the filename be constant? the copied files are independent of host :thinking:

Reply:
I'm going to change it to
- Copy scripts => constant filename
- Configure ssh => use `gitpodHost`